projects/js-packages/publicize-components/src/components/media-picker/styles.module.scss

```diff
@@ -72,7 +72,7 @@
 	cursor: pointer;
 }
 
-.remove {
+%remove {
```

Comment:
This one probably has breakage too, like the boost thing below.

Reply:
e0f7bf441a